hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/RSRpcServices.java

```diff
@@ -3946,4 +3949,19 @@ public GetCachedFilesListResponse getCachedFilesList(RpcController controller,
     });
     return responseBuilder.addAllCachedFiles(fullyCachedFiles).build();
   }
+
+  @Override
+  public CleanBucketCacheResponse cleanBucketCache(RpcController controller,
```

Comment:
I'm confused why we are implementing the eviction logic here. The RSRpcServices should be just a facade, finding "orphans" blocks and evict it should be a responsibility of the cache implementation class.